app/views/HomePageView.scala.html

```diff
@@ -14,25 +14,27 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.components._
+
 @this(
-        layout: templates.Layout,
-        govukNotificationBanner: GovukNotificationBanner
+layout: templates.Layout,
+govukPhaseBanner: GovukPhaseBanner,
+notificationBanner: notificationBanner,
+paragraphBody: paragraphBody
 )
 
-@(downloadReady: Boolean = false)(implicit request: Request[_], messages: Messages)
+@(downloadReady: Boolean)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = s"${messages("homepage.title")} - ${messages("site.govuk")}", showBackLink = false, headerShouldHaveServiceLink = false) {
 
     @if(downloadReady) {
-        @govukNotificationBanner(NotificationBanner(
```

Comment:
is there any reason why we are not just using the govukNotificationBanner?

Reply:
@I have changed it back, I could not find the way to change the colour from green to blue as changes were not in plain sight so I used GovUk components. However I have now resolved this as per comments.

Reply:
excellent - where possible always use govuk components as accessibility etc will be fully tested and they will be the accepted standard :)